cmd/telegraf/agent.conf

```diff
@@ -53,13 +53,13 @@
   ## Log only error level messages.
   # quiet = false
 
-  ## Log target controls the destination for logs and can be one of "file",
-  ## "stderr" or, on Windows, "eventlog".  When set to "file", the output file
-  ## is determined by the "logfile" setting.
+  ## Log target controls the destination for logs and can be one of "text",
+  ## "structured" or, on Windows, "eventlog". The output file (if any) is
```

Comment:
They become deprecated. The current settings are redundant and are prone to be contradictory, e.g. `logtarget="stderr"` but a `logfile` is set... Or even worst `logtarget="file"` but `logfile` is empty...

Reply:
My only concern is around deprecating a default value that's been there for a while might be a little concerning. Right this would touch pretty much everyone's config.

Is it worth the churn? If you think so, then I'm good with this.